gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/base.py.j2

```diff
@@ -43,15 +57,17 @@ class {{ service.name }}Transport(abc.ABC):
         {%- endfor %}
     )
 
+    DEFAULT_HOST: str = {% if service.host %}'{{ service.host }}'{% else %}{{ '' }}{% endif %}
```

Comment:
@bshaffer Are there any concerns with making DEFAULT_HOST a public attribute?

Reply:
I don't believe there is any issue there!